test/crawlers/cheerio_crawler.test.js

```diff
@@ -134,8 +136,6 @@ describe('CheerioCrawler', () => {
 
         processed.forEach((request) => {
             expect(request.userData.title).toBe('Title');
-            expect(typeof request.userData.html).toBe('string');
-            expect(request.userData.html.length).not.toBe(0);
```

Comment:
If you're deleting it here, would make sense to delete it also from the handle page function or replace it with something else.